apps/deploy.mdx

```diff
@@ -9,12 +9,40 @@ Once you deploy an app on Kernel, you can schedule its actions on a job or run t
 
 ## Deploy the app
 
+### From local directory
+
 Use our CLI from the root directory of your project:
 ```bash
-# entrypoint_file_name should be where you've defined your Kernel app
 kernel deploy <entrypoint_file_name>
 ```
 
+#### Notes
+
+- The `entrypoint_file_name` is the file name where you [defined](/apps/develop) your app.
+- Include a `.gitignore` file to exclude dependency folders like `node_modules` and `.venv`.
+
+### From GitHub
+
+You can deploy an app directly from a public or private GitHub repository using the Kernel CLI — no need to clone or manually push code.
+
+```bash
+kernel deploy github \
+  --url https://github.com/<owner>/<repo> \
+  --ref <branch|tag|commit> \
+  --entrypoint <path/to/entrypoint> \
+  [--path <optional/subdir>] \
+  [--github-token <token>] \
+  [--env KEY=value ...] \
+  [--env-file .env] \
+  [--version latest] \
+  [--force]
+```
+
+#### Notes
+- **`--path` vs `--entrypoint`:** Use `--path` to specify a subdirectory within the repo (useful for monorepos), and `--entrypoint` for the path to your app's entry file relative to that directory (or repo root if no `--path` is specified).
```

Comment:
Does this apply to local directory deployment too? If so, I would suggest moving it to the Deployment Notes section that is more general.